Templates.Settings.targets

```diff
@@ -60,20 +60,15 @@
     
     <Target Name="DeterminePackageVersions" DependsOnTargets="SetupPackageSource">
         <ItemGroup Condition="'$(CoherencePackageSources)' != ''">
-            <CoreAppFile Include="$(PackageSourceExternal)\Microsoft.NETCore.App.*.nupkg"/>
```

Comment:
Nope, since we aren't picking up semver'd versions of this, but instead the static version, we just set the version to 1.0.3 directly.. see line 99